src/client/components/breadcrumbs/index.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Breadcrumb } from 'semantic-ui-react';
+import './index.scss';
+
+class Breadcrumbs extends React.Component {
+    static defaultProps = {
+        sections : [
```

Comment:
Sample data and default props are different things. It makes sense to use a default props like `content: ''` (empty string) or `content: 'no content'` (placeholder). Don't every add sample data as default props.